CONTRIBUTING.md

```diff
@@ -44,9 +44,9 @@ If you get a permissions error, then try re-running the command with `sudo`.
 
 7. Learn the essentials of our [Docs Structure](#docs-structure).
 
-8. Review our simple [Style Guide](https://github.com/cockroachdb/docs/wiki/Style-Guide).
+8. Review our simple [Style Guide](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md.
```

Comment:
I'd remove the word "simple" while we're here, if you agree.

Reply:
Definitely.